addons/mail/static/src/messaging/component/discuss/discuss_tests.js

```diff
@@ -4615,24 +4625,24 @@ QUnit.test('moderation: moderated channel with pending moderation message', asyn
     );
     assert.isNotVisible(
         document.querySelector('.o_widget_Discuss_controlPanelButtonModeration.o-accept'),
-        'the moderate button "Accept" should be invisible by default'
+        "the moderate button 'Accept' should be invisible by default"
     );
     assert.containsOnce(
         document.body,
         '.o_widget_Discuss_controlPanelButtonModeration.o-reject',
         "there should one moderate button to reject messages pending moderation"
     );
     assert.isNotVisible(
-        '.o_widget_Discuss_controlPanelButtonModeration.o-reject',
-        'the moderate button "Reject" should be invisible by default'
+        document.querySelector('.o_widget_Discuss_controlPanelButtonModeration.o-reject'),
+        "the moderate button 'Reject' should be invisible by default"
```

Comment:
prep + why add the querySelector if it wasn't needed before?

Reply:
mistake of prep